tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -638,22 +641,22 @@ public function test_process_directives_changes_html_if_contains_svgs() {
 		);
 		$html           = '
 			<header>
-				<svg height="100" data-wp-bind--width="myPlugin::state.width">
+				<div data-wp-bind--id="myPlugin::state.width"></div>
+				<svg height="100">
 					<title>Red Circle</title>
 					<circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
 				</svg>
 				<div data-wp-bind--id="myPlugin::state.id"></div>
-				<div data-wp-bind--id="myPlugin::state.width"></div>
 			</header>
 		';
 		$processed_html = $this->interactivity->process_directives( $html );
 		$p              = new WP_HTML_Tag_Processor( $processed_html );
+		$p->next_tag( 'div' );
+		$this->assertEquals( '100', $p->get_attribute( 'id' ) );
 		$p->next_tag( 'svg' );
 		$this->assertNull( $p->get_attribute( 'width' ) );
```

Comment:
This assertion doesn't make much sense now. Maybe we should keep the `data-wp-bind--width` on the SVG? It seems like this was intended to assert that the directive wasn't processed on the SVG, but asserting that an attribute that doesn't have any reason to exist is odd.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6413/commits/415492536fed71329c0280ab94911ea938e6eff6